app/aem/core/src/main/java/com/cognifide/apm/core/progress/ProgressImpl.java

```diff
@@ -131,8 +132,13 @@ public String getExecutor() {
     return executor;
   }
 
+  @Override
+  public Calendar getStartTime() {
+    return startTime;
+  }
+
   @Override
   public long determineExecutionDuration() {

```

Comment:
opt-in / if you could use more dedicated types like https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html